include/sys/statvfs.h

```diff
@@ -49,7 +49,14 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-/* struct statfs file system types. */
+/* Definitions for the flag in `f_flag'.  These definitions should be
+ * kept in sync with the definitions in <sys/mount.h>.
+ */
+
+#define ST_RDONLY             0x0001 /* Mount read-only.  */
+#define ST_NOSUID             0x0002 /* Ignore suid and sgid bits.  */
```

Comment:
we don't have NOSUID things.

Reply:
Yes, but it isn't harmful to keep it here since we may imlement this feature someday.

Reply:
ok